### ISSUE TITLE
[Bug]: Repeating error while using Honey Generator UI

### ISSUE THREAD

**Issue by talrey** [Resourceful-Bees/ResourcefulBees#278]
### Bug Description

I'm seeing this error happening constantly while using the Honey Generator's UI in a dev environment.

The generator and its UI seem to still be working, but filling up the console with this doesn't seem ideal.

`java.lang.IndexOutOfBoundsException: readerIndex(30) + length(4) exceeds writerIndex(30): UnpooledByteBufAllocator$InstrumentedUnpooledUnsafeHeapByteBuf(ridx: 30, widx: 30, cap: 256)
	at io.netty.buffer.AbstractByteBuf.checkReadableBytes0(AbstractByteBuf.java:1405) ~[netty-all-4.1.25.Final.jar:4.1.25.Final] {}
	at io.netty.buffer.AbstractByteBuf.readInt(AbstractByteBuf.java:786) ~[netty-all-4.1.25.Final.jar:4.1.25.Final] {}
	at net.minecraft.network.PacketBuffer.readInt(PacketBuffer.java:789) ~[forge-1.16.5-36.2.1_mapped_official_1.16.5-recomp.jar:?] {re:classloading}
	at com.teamresourceful.resourcefulbees.common.tileentity.HoneyGeneratorTileEntity.handleGUINetworkPacket(HoneyGeneratorTileEntity.java:235) ~[main/:?] {re:classloading}
	at com.teamresourceful.resourcefulbees.common.network.packets.SyncGUIMessage.lambda$handle$0(SyncGUIMessage.java:37) ~[main/:?] {re:classloading}
	at net.minecraftforge.fml.network.NetworkEvent$Context.enqueueWork(NetworkEvent.java:215) ~[forge-1.16.5-36.2.1_mapped_official_1.16.5-recomp.jar:?] {re:classloading}
	at com.teamresourceful.resourcefulbees.common.network.packets.SyncGUIMessage.handle(SyncGUIMessage.java:32) ~[main/:?] {re:classloading}
`

(shortened for conciseness and because it repeats. I can attach the full file if necessary...)

### How to Reproduce?

1. Launch the mod in a dev environment
2. Place a Honey Generator
3. Open the UI
4. Observe the console / log

### Version

1.0.0-alpha

### Mod Loader Version

1.16.5 - 36.2.1

### Acknowledgement

- [X] I agree that I read the terms at the top of the page.

**Comment by Epic428** [comment]
Why are you using 1.0.0-alpha? We don’t even have a jar built for it…. Also I’m pretty sure I know what the issue is and how to fix it

**Comment by ThatGravyBoat** [comment]
The generator will be getting rewrote when I get a chance to so this won't be an issue when that gets done.

**Comment by talrey** [comment]
Glad to hear you found the fix. To be honest, I was only using it (and in a dev environment besides) because I'd heard there was a compat issue with Create and I wanted to investigate that. Turns out there's a Create issue regarding fluid handler blocks that can be Waterlogged, so that solved that mystery.

I just noticed this while I was testing the fluid issue, if it's getting rewritten then go ahead and close.
